hpc0.qmd

```diff
@@ -136,14 +134,11 @@ Two new commands:
 
 ### Solution
 
-1:
-
-```{.bash code-line-numbers=false}
+1. ```{.bash code-line-numbers=false}
```

Comment:
This solution turned out really good. I confess that I doubted it would work at first.